client/src/app/infrastructure/utils/validators/is-number-range.ts

```diff
@@ -1,11 +1,11 @@
 import { AbstractControl, ValidatorFn } from '@angular/forms';
 
-export function isNumberRange(minCtrlName: string, maxCtrlName: string): ValidatorFn {
+export function isNumberRange(minCtrlName: string, maxCtrlName: string, errorName: string): ValidatorFn {
```

Comment:
Might be a good idea to set a default value for that. 

```suggestion
export function isNumberRange(minCtrlName: string, maxCtrlName: string, errorName: string = `rangeError`): ValidatorFn {
```